### ISSUE TITLE
expected order of processes in process_tree

### ISSUE THREAD

**Issue by k4otix** [microsoft/msticpy#500]
**Describe the bug**
I've noticed that processes in `process_tree` are not displayed in chronological order (using a custom `ProcSchema` for Linux events captured from eBPF)

**To Reproduce**
1. Load the attached CSV
2. Print the dataframe
3. create/plot process_tree
4. observe some process events are not in the expected order

**Expected behavior**
I would expect each level of the process tree to be sorted chronologically. For example, my `uname -a` command (PID 13575, which is properly shown as a child process of my bash shell) comes in the middle of other events that occur when I SSH'd to a remote system, instead of at the end. There are other examples in the output as well that are out of order based on PID and timestamp.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS 12.5.1
 - Browser: N/A - Jupyter notebook in Visual Studio Code
 - msticpy version: 2.0.0, python version 3.10.6

**Additional context**

**Notebook Code:**
Initialize/Read data
```python
import pandas as pd
from msticpy.transform.proc_tree_schema import ProcSchema
from msticpy.vis import process_tree

pd.set_option('display.max_colwidth', None)

schema = {
    'time_stamp': 'ts',
    'process_name': 'process_name',
    'process_id': 'Pid',
    'parent_name': None,
    'parent_id': 'Ppid',
    'logon_id': 'logon_id',
    'target_logon_id': None,
    'cmd_line': 'Argv',
    'user_name': 'Username',
    'user_id': 'Uid',
    'host_name_column': 'host_s',
    'path_separator': '/'
}
schema = ProcSchema(**schema)

df = pd.read_csv('test.csv')
df['ts'] = pd.to_datetime(df['Timestamp [UTC]'])
df['process_name'] = df['Argv'].str.split(' ').str[1]
df['logon_id'] = 1  # not in the source data; needed a value for session ID for plot to work
df.tail(n=1)
```

| Timestamp [UTC] | host_s | Uid | Username | Pid | Ppid | Argv | ts | process_name | logon_id
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- 
9/9/2022, 5:05:08.499 PM | example-01.example.com | 59159 | testuser1 | 13575 | 13153 | /usr/bin/uname -a | 2022-09-09 17:05:08.499 | /usr/bin/uname | 1

Plot process_tree
```python
process_tree.plot_process_tree(data=df, schema=schema, show_table=True, pid_fmt='dec')
```
<img width="587" alt="Screen Shot 2022-09-12 at 1 21 25 AM" src="https://user-images.githubusercontent.com/10135730/189579178-22f65172-7fe4-46fe-89d9-36966fc95193.png">

Test file:
[test.csv](https://github.com/microsoft/msticpy/files/9544789/test.csv)

**Comment by ianhelle** [comment]
I'm amazed that nobody (including me) has found/reported this before. I need to do another fix on this code so fix this at the same time.
thx for reporting

**Comment by ianhelle** [comment]
Fixed in #513
